### ISSUE TITLE
No result whith CP/Jackett

### ISSUE THREAD

**Issue by se7ensinsman** [Jackett/Jackett#1610]
Hi everyone !
I’ve got the last version (0.7.1622) and I have no result with torrentpotato. the manual search is working and send me many results.
Here is the request sent by CP : http://127.0.0.1:9117/potato/torrent9?imdbid=tt4979860&user=&passkey=xxx
All my settings seem to be good with the API in passkey and the user leaved with a blank.
Any ideas ? Thanks for your help !
Benjamin

**Comment by zogtar** [comment]
+1
Hello,
I have the same issue. 

jackett see the CP request but dont find any release :
"Found 0 torrentpotato releases from YGGtorrent for: XXX"
A manual search work very well :
"Manual search for "XXX" on YGGtorrent with 15 results."

config : synocommunity package (CP & jackett).
tracker used : nexttorrent, yggtorrent, ...

It seems to be the same issue.

**Comment by DocteurB** [comment]
need a new feature on jackett to edit provider who dosn't word with imdbid (to ignore the filter )
or
need a new feature on couchpotato to ignore filter imdbid.

**Comment by zogtar** [comment]
Sorry DocteurB, i don't understand your answer.
With so little information, your conclusion is so fast.

**Comment by DocteurB** [comment]
i try : http://my.ip:9117/potato/yggtorrent?imdbid=tt0490215&search=Silence+2016&user=&passkey=xxx
{
  "results": [],
  "total_results": 0
}

and i try : http://my.ip:9117/potato/yggtorrent?&search=Silence+2016&user=&passkey=xxx
{
  "results": [des trucs
    {
.....
......
  "total_results": 6
}

so my gess was that YGGtorrent (like torrent9) dosn't work with imdbid , and all the request from CP comme with this key word.

the link imdb is done by Jackett or it's an information given by the indexer ?

if it's the first choice i need to understand why it dosn't work with the movies from yggtoerent and torrent9.

**Comment by cloneMe** [comment]
To my mind, this issue duplicates this one: TorrentPotato doesn't find torrents #1606

**Comment by flightlevel** [comment]
Duplicate https://github.com/Jackett/Jackett/issues/1606
